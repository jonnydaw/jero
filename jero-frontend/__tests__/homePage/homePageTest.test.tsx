import '@testing-library/jest-dom'
import { render, screen } from '@testing-library/react'
import {useTranslations} from 'next-intl';
import { NextIntlClientProvider } from 'next-intl';

import HomePage from '../../app/[locale]/page'
// https://github.com/amannn/next-intl/discussions/879
const locale = "en";
// const messages = useTranslations()

describe('Home Page', () => {
  it('should render correctly', () => {
    const { container } = render(
      <NextIntlClientProvider locale={locale}>
        <HomePage/>
      </NextIntlClientProvider>,
    );
    const heading = screen.getByRole('heading', { level: 1 })
 
    expect(heading).toBeInTheDocument()
  });
});