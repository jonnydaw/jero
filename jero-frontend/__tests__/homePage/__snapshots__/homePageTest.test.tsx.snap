// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Home Page should render correctly 1`] = `
<div>
  <div>
    <h1>
      HomePage.title
    </h1>
    <a
      href="/en/about"
    >
      HomePage.about
    </a>
  </div>
</div>
`;
